README.md

```diff
@@ -57,6 +57,7 @@ Ethereum chains support the following additional options:
 contract = ""0x12345..." // The address of the bridge contract
 gasPrice = "0x1234"      // Gas price for transactions 
 gasLimit = "0x1234"      // Gas limit for transactions
+connectionType = "rpc"  // The type of connection for the ethereum chain. The options are RPC and HTTP. If left blank, the default is RPC.
```

Comment:
This isn't really what you've implemented. Anything except 'http' will be considered 'rpc' at this point. Also the options would be 'http' and 'ws', these both use rpc. Please update this and the parsing to ensure they match.

The way I initially pictured this was to have an `http` boolean option, much like in `ethereum.Connection`. Perhaps this would simplify things a little.